docker/compose/faucet-validator-fullnode/local_client.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# Copyright (c) The Diem Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+set -e
+
+DIR="$( cd "$( dirname "$0" )" && pwd )"
+
+export WAYPOINT=$(cat "$DIR"/validator_data/waypoint.txt)
+echo "Waypoint: $WAYPOINT"
+
+docker run --network diem-docker-compose-shared \
+          -it libra/client:devnet \
+          --url http://172.16.1.3:8080 \
+          --faucet-url http://172.16.1.2:8000 \
+          --chain-id TESTING \
+          --waypoint "$WAYPOINT"
```

Comment:
is this temporary? or will it also be in compose?

Reply:
there's no real clean way to get a real shell via docker compose (i.e `docker run -it ...`)

Reply:
oh, I thought if I just did `docker-compose up` it would leave one of them as the foreground... is that possible? If not, I would prefer we avoid writing a shell script and just shove this command into a readme (or both?). Since it is technically platform independent.

Reply:
can we make it explicitly clear which platforms we support? I.e is windows a "best effort" or do we intend to actively support it?